tests/publiccloud/storage_perf.pm

```diff
@@ -99,19 +100,20 @@ sub analyze_previous_series {
         my $previous_records_mean = get_mean_from_db($args, " limit 5 offset 5");
         # we do analysis only if we get back some non-zero values
         if (defined($previous_records_mean) && defined($last_records_mean) && $previous_records_mean != 0 && $last_records_mean != 0) {
-            record_info('ANALYZE', "Checking " . $current_load_type);
-            # this formula detect if mean values differs more than 10%
-            if ((abs($previous_records_mean - $last_records_mean) / $previous_records_mean) * 100 > 10) {
-                die sprintf("Anomaly occurred in http://openqa-perf.qa.suse.de/d/hoRc37HWz/storage-performance?orgId=1&var-os_flavor=%s&var-os_version=%s", $args->{os_flavor}, $args->{os_flavor});
+            my $diff_percents = (abs($previous_records_mean - $last_records_mean) / $previous_records_mean) * 100;
```

Comment:
I like more https://en.wikipedia.org/wiki/Moving_average

Reply:
but it's ok

Reply:
this discussion anyway goes a bit beyond what Pavel is trying to achieve here . @ilausuch feel free to create a ticket if you still thinks that it make sense to do any changes in this direction.